.github/workflows/deployRelease.yml

```diff
@@ -18,6 +18,18 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           version: ${{ github.event.inputs.version }}
+  createTestBranch:
+    name: Create test branch from release tag
+    runs-on: ubuntu-latest
+    needs: validate
```

Comment:
This should be `needs: release`, and then move this job below release for consistency.